pkg/cli/gc.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/gadget-inc/dateilager/internal/key"
+	"github.com/gadget-inc/dateilager/internal/logger"
+	"github.com/gadget-inc/dateilager/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdGc() *cobra.Command {
+	var (
+		mode    string
+		project int64
+		keep    int64
+		from    *int64
+		sample  float32
+	)
+
+	cmd := &cobra.Command{
+		Use: "gc",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := cmd.Context()
+			c := client.FromContext(ctx)
+
+			var count int64
+			var err error
+
+			switch mode {
+			case "contents":
+				if sample == -1 {
+					return fmt.Errorf("--sample required for contents mode")
+				}
+
+				count, err = c.GcContents(ctx, sample)
+				if err != nil {
+					return fmt.Errorf("could not gc contents: %w", err)
+				}
+			case "project":
+				if project == -1 {
+					return fmt.Errorf("--project required for project mode")
+				}
+				if keep == -1 {
+					return fmt.Errorf("--keep required for project mode")
+				}
+				if *from == -1 {
+					from = nil
+				}
+
+				count, err = c.GcProject(ctx, project, keep, from)
+				if err != nil {
+					return fmt.Errorf("could not gc project %v: %w", project, err)
+				}
+			case "random-project":
```

Comment:
Can we make this plural so that it's clear it's going to gc multiple projects?

```suggestion
			case "random-projects":
```